src/main/java/com/monumental/services/MonumentService.java

```diff
@@ -42,19 +53,17 @@ private void buildFTSQuery(CriteriaBuilder builder, CriteriaQuery query, Root ro
         CriteriaQuery<Monument> query = this.createCriteriaQuery(builder, false);
```

Comment:
The comment above the `search` method should be fixed.

Reply:
Please :)

Reply:
@njd9419